src/utils/CanvasTemplates.js

```diff
@@ -490,4 +490,143 @@ module.exports = class CanvasTemplates {
+      const MEME = Math.random() > 0.99 && '"Comic Sans MS"'
+      const EXTRABOLD = MEME || '"Montserrat ExtraBold"'
+      const REGULAR = MEME || '"Montserrat"'
+      const LIGHT = MEME || '"Montserrat Light"'
+      return {
+        TITLE: `17px ${EXTRABOLD}`,
+        TEMPERATURE: `bold 90px ${EXTRABOLD}`,
+        INFORMATIONS: `17px ${LIGHT}`,
+        WEEK_DAYS: `17px ${REGULAR}`,
+        WEEK_TEMPERATURES: `bold 29px ${LIGHT}`
+      }
+    })()
+
+    const IMAGE_ASSETS = Promise.all([
+      Image.from(Constants.DEFAULT_BACKGROUND_GRAY_PNG, true),
+      Image.buffer(Constants.ARROW_UP_SVG, true),
+      Image.buffer(Constants.WIND_SVG, true)
+    ])
+
+    const WEATHER_ICONS = Promise.all([
+      Image.buffer(Constants.WEATHER_CLEAR_DAY, true),
+      Image.buffer(Constants.WEATHER_CLEAR_NIGHT, true),
+      Image.buffer(Constants.WEATHER_CLOUDY, true),
+      Image.buffer(Constants.WEATHER_FOG, true),
+      Image.buffer(Constants.WEATHER_PARTLY_CLOUDY_DAY, true),
+      Image.buffer(Constants.WEATHER_PARTLY_CLOUDY_NIGHT, true),
+      Image.buffer(Constants.WEATHER_RAIN, true),
+      Image.buffer(Constants.WEATHER_SNOW, true),
+      Image.buffer(Constants.WEATHER_THUNDERSTORM, true),
+      Image.buffer(Constants.WEATHER_WIND, true)
+    ])
+
+    const canvas = createCanvas(WIDTH, HEIGHT)
+    const ctx = canvas.getContext('2d')
+
+    // CARD DRAWING
+    ctx.fillStyle = 'rgba(0, 0, 0, 0.7)'
+    ctx.fillRect(CARD_X_MARGIN, CARD_Y_MARGIN, CARD_WIDTH, CARD_HEIGHT)
+
+    const BRIGHTER_HEIGHT = 150
+    ctx.fillStyle = 'rgba(255, 255, 255, 0.1)'
+    ctx.fillRect(CARD_X_MARGIN, CARD_Y_MARGIN, CARD_WIDTH, BRIGHTER_HEIGHT)
+
+    // MAIN TEMPERATURE
+    const MAIN_TEMPERATURE_Y = 150
+    ctx.fillStyle = '#FFFFFF'
+    ctx.write(weather.now.temperature, CARD_X_MARGIN * 1.5, MAIN_TEMPERATURE_Y, FONTS.TEMPERATURE, ALIGN.BOTTOM_LEFT)
+
+    const DAILY_MARGIN = CARD_X_MARGIN + 35
+    const DAYS_SEPARATOR = 63
+    const DAILY_Y = CARD_Y_MARGIN + BRIGHTER_HEIGHT + 20
+    const WEATHER_ICONS_SIZE = 40
+
+    // MAIN INFORMATIONS
+
+    const MAIN_INFORMATIONS_Y = 180
+    const MAIN_INFORMATIONS_X_MARGIN = 35
+    const MAIN_INFORMATIONS_X_MARGIN_BETWEEN = 30
+
+    ctx.write(weather.now.max, CARD_X_MARGIN + MAIN_INFORMATIONS_X_MARGIN, MAIN_INFORMATIONS_Y, FONTS.INFORMATIONS, ALIGN.BOTTOM_LEFT)
+    const MAIN_INFORMATIONS_MAX_W = measureText(ctx, FONTS.INFORMATIONS, weather.now.max).width
+    ctx.write(weather.now.min, CARD_X_MARGIN + MAIN_INFORMATIONS_X_MARGIN + MAIN_INFORMATIONS_MAX_W + MAIN_INFORMATIONS_X_MARGIN_BETWEEN, MAIN_INFORMATIONS_Y, FONTS.INFORMATIONS, ALIGN.BOTTOM_LEFT)
+    const MAIN_INFORMATIONS_MIN_W = measureText(ctx, FONTS.INFORMATIONS, weather.now.min).width
+    ctx.write(weather.now.wind, CARD_X_MARGIN + MAIN_INFORMATIONS_X_MARGIN + MAIN_INFORMATIONS_MAX_W + MAIN_INFORMATIONS_MIN_W + MAIN_INFORMATIONS_X_MARGIN_BETWEEN * 2, MAIN_INFORMATIONS_Y, FONTS.INFORMATIONS, ALIGN.BOTTOM_LEFT)
```

Comment:
`ctx.write` returns some useful information, you should use it instead of using `measureText` every single time.